app/bundles/PointBundle/Tests/Unit/Model/PointModelTest.php

```diff
@@ -0,0 +1,142 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Mautic\PointBundle\Tests\Unit\Model;
+
+use Doctrine\ORM\EntityManager;
+/**
+ * @deprecated to be removed in 3.0
+ */
+use Mautic\CoreBundle\Factory\MauticFactory;
```

Comment:
Not sure why this is here
```suggestion
use Mautic\CoreBundle\Factory\MauticFactory;
```